src/Hakyll/ShortcutLinks.hs

```diff
@@ -10,6 +10,45 @@ package in websites generated by [hakyll](https://hackage.haskell.org/package/ha
 
 The flexible interface allows to use the supported huge collection of shortcuts
 along with using custom ones.
+
+Here is a few examples of the `@github` shortcut :
+
+
+- Link to a user :
+
+  @
+  [kowainik](@github:kowainik)
+  [kowainik](https://github.com/kowainik)
+  @
+
+  @
+  [kowainik](@github(kowainik))
+  [kowainik](https://github.com/kowainik)
```

Comment:
Btw, Haddock also supports tables. Unfortunately, with different syntax. But it would be nice to have tables in Haddock as well. You can find an example of such a table in the following module:

* https://hackage.haskell.org/package/slist-0.0.0/docs/Slist.html